### ISSUE TITLE
[FEATURE] Reverse tabstop

### ISSUE THREAD

**Issue by half-adder** [artisticat1/obsidian-latex-suite#228]
# Description of the Problem
On web forms, you can hold shift while pressing tab to move in reverse through web forms. Right now, we can only move forward in the equation, it would be nice if we could move back to the last tabstop and change what we entered.

# Description of the Solution
Shift+Tab should allow for backwards tabstop movement.

# Additional
Could be tricky since the actual tabstop characters (`$0`, ...) are now gone. Would have to remember their place in the string somehow.

**Comment by artisticat1** [comment]
I agree this would be useful to have. Should be possible using CodeMirror's undo `history` extension.
